app/frontend/src/App.svelte

```diff
@@ -1,56 +1,140 @@
 <script lang="ts">
-  import { Init } from "./lib/state.svelte"
-  import TreeNodeView from "./lib/components/TreeNode.svelte"
-  import { deviceTree, deviceLogs } from "./lib/state.svelte"
-  import { Refresh } from "../wailsjs/go/main/App"
-  Init()
+  import { onMount } from "svelte"
+  import { Content } from "carbon-components-svelte"
+
+  import { Init } from "$lib/state.svelte"
+  import Header from "$lib/components/Header.svelte"
+  import Tree from "$lib/components/Tree.svelte"
+  import Log from "$lib/components/Log.svelte"
+  import Tooltip from "$lib/components/Tooltip.svelte"
+  import {
+    handlePointerDown,
+    handleWindowPointerMove,
+    handleWindowPointerUp,
+    isResizing,
+    setContentRegion,
+    treeRatio,
+  } from "$lib/split-layout"
+
+  let contentRegion: HTMLDivElement | null = null
```

Comment:
```suggestion
  let contentRegion: HTMLDivElement | undefined = undefined
```

Generally good idea to avoid `null` in JavaScript and use `undefined` instead, they behave quite differently in a lot of .. interesting situations.

Reply:
doing so breaks the next line 

<img width="1516" height="229" alt="image" src="https://github.com/user-attachments/assets/aa77b5bf-825e-480b-af4b-534a0915958b" />

Reply:
changed to just equal the divelement